backend/database/database.go

```diff
@@ -157,6 +157,14 @@ func SetupTables(db *sqlx.DB) error {
 			FOREIGN KEY (UserID) REFERENCES User(ID) ON DELETE CASCADE ON UPDATE CASCADE
 		)
 		`,
+		`CREATE TABLE PastSearch (
+			UserID CHAR(36) NOT NULL,
+			CuisineTypes VARCHAR(36) NOT NULL,
+			RelevantSearchWord VARCHAR(255) NOT NULL,
+			PRIMARY KEY (UserID),
```

Comment:
If `UserID` is not unique so it cannot be the PK. A user can create multiple records.